packages/app/src/components/common/entities/WebViewWithInjected/hooks/useMiddleware.ts

```
@@ -0,0 +1,84 @@
+
+    // 라우팅 메시지 처리
+    if (name === ("route-to" as string) && method === "POST") {
+      const { path, routeType, params } = body as {
+        path: string;
+        routeType?: "replace" | "push";
+        params?: Record<string, any>[];
+      };
+
+      routeType === "replace"
+        ? router.replace(getPathToRoute({ path, params }))
+        : router.push(getPathToRoute({ path, params }));
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)